src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -527,7 +527,10 @@ describe('XhrPlugin tests', () => {
         plugin.disable();
 
         // Assert
-        expect(record).not.toHaveBeenCalled();
```

Comment:
question: did this change due to the default behavior change?

Reply:
Sorry for the late response. `record(type, event)` can record any event, including both XrayTrace and HttpEvent. Here, I expect an HttpEvent to be recorded because it is enabled by default, even though trace is disabled. I do not expect `record(XRAY_TRACE_EVENT_TYPE, ...)` to be recorded